### PR TITLE
fixed remote write regression

### PR DESCRIPTION
**Pull request by bwplotka**
We forgot to add it to the run group.

Fixes https://github.com/prometheus-community/avalanche/issues/86

Chained on top of https://github.com/prometheus-community/avalanche/pull/87

### CODE REVIEW

File: cmd/avalanche.go [hunk 1]
```diff
@@ -147,18 +147,19 @@ func main() {
...
-		}
-		if *remotePprofInterval > 0 {
-			done <- struct{}{}
-			wg.Wait()
-		}
-		return
+		ctx, cancel := context.WithCancel(context.Background())
+		g.Add(func() error {
+			if err := metrics.SendRemoteWrite(ctx, config, reg); err != nil {
+				log.Fatal(err)
```
Comment (MichaHoffmann) [inline cmd/avalanche.go:153]: it feels like it should pass error to parent and then we can fatal there, that would at least cancel the context, not sure if imporatnt though

**Review by MichaHoffmann — APPROVED** [review]
lgtm

File: cmd/avalanche.go (hunk 1 above)
Comment (bwplotka) [reply cmd/avalanche.go:153]: ups, yup!

Comment (MichaHoffmann) [reply cmd/avalanche.go:153]: Fatal is called by g.Run already even, so just passing error is enough

**Pull request merged**